src/renderer/dx/DxRenderer.cpp

```diff
@@ -61,7 +61,6 @@ using namespace Microsoft::Console::Types;
 // TODO GH 2683: The default constructor should not throw.
 DxEngine::DxEngine() :
     RenderEngineBase(),
-    _invalidateFullRows{ true },

```

Comment:
Why did you pull this? I presume the answer is:
1. No one ever set this so we'll always just invalidate a full row.
2. We determined that we likely won't even need to do differential drawing with a better render strategy anyway so there's no reason to try to tighten down the differential drawing any further in the future by doing partial rows.
If that's the answer... maybe leave some of that in a comment in `InvalidateRectangle` so we don't forget.

Reply:
Yeah basically that. I was touching the only function that uses this field and I doubt this field will ever be useful again, as it's slower to figure out what contains ligatures, compared to just redrawing the entire row. The difference between drawing no text and a full row of text is only 5-10μs after all. (Also: YAGNI. 😄)

Reply:
Fair enough.